File: src/main/java/org/mystic/jetbrains/challenge/alpha/Sum.kt
package org.mystic.jetbrains.challenge.alpha

import java.util.Scanner
import java.io.PrintWriter

fun main(args: Array<String>) {
    val sc = Scanner(System.`in`)
    val a = sc.nextInt()
    val b = sc.nextInt()
    sc.close()
    val out = PrintWriter(System.out)
    out.println(a + b)
    out.close()
}

File: src/main/java/org/mystic/acm/timus/Sum.scala
package org.mystic.acm.timus

import java.lang.String
import java.io.{BufferedOutputStream, PrintWriter, InputStreamReader, BufferedReader}
import java.util.StringTokenizer

object Sum {

  var out: PrintWriter = _
  var br: BufferedReader = _
  var st: StringTokenizer = _

  def next: String = {
    while (st == null || !st.hasMoreTokens) {
      st = new StringTokenizer(br.readLine)
    }
    st.nextToken
  }

  def nextInt: Int = {
    Integer.parseInt(next)
  }

  def solve: Unit = {
    out.println(nextInt + nextInt)
  }


  def main(args: Array[String]): Unit = {
    br = new BufferedReader(new InputStreamReader(System.in))
    out = new PrintWriter(new BufferedOutputStream(System.out))
    solve
    out.close
  }
}
